### DISCUSSION TITLE
The image processing effect of Linux executable files is inconsistent with that of Windows installation version

### DISCUSSION THREAD

**Discussion by zxbzxb180** [ImageMagick/ImageMagick#7651] [Development]
### ImageMagick version

7.1.1.-38

### Operating system, version and so on

Win10、CentOS

### Description

commander: convert input.gif -coalesce output.gif

Windows:
![image](https://github.com/user-attachments/assets/a0d8be68-28c1-4cdc-90da-d38bdb52a9af)
output GIF is normal

CentOS:
![image](https://github.com/user-attachments/assets/91f3dd40-885e-495d-adab-e7390133d4cf)
The GIF shows a blurred screen，like:
![image](https://github.com/user-attachments/assets/8900bbfd-baaf-4f74-987d-02fd75d83584)

**Comment by dlemstra** [comment]
Without the input file this would be a lot of guessing.

**Comment by snibgo** [comment]
@zxbzxb180: Please show your input GIF, and the outputs from each system. Show the actual GIFs, not screenshots.

`convert` is deprecated. Use `magick` instead.
